binding/java/src/ai/picovoice/rhino/Utils.java

```diff
@@ -1,5 +1,5 @@
 /*
-    Copyright 2018-2020 Picovoice Inc.
+    Copyright 2018-2021 Picovoice Inc.
```

Comment:
I notice in this file there's a `getTestContextPath` function that probably should just be moved to RhinoTest. Seems weird to include a test function in our binding Utils file